src/main/java/core/basesyntax/Calculator.java

```
@@ -0,0 +1,74 @@
+package core.basesyntax;
+
+public class Calculator {
+    private Integer result;
+    private Integer firstNumber;
+    private Integer secondNumber;
+
+    public Calculator(Integer firstNumber, Integer secondNumber) {
+        this.firstNumber = firstNumber;
+        this.secondNumber = secondNumber;
+    }
+
+    public boolean checkForNull() {
+        if (firstNumber == null || secondNumber == null) {
+            throw new NullPointerException();
+        }
+        return false;
+    }
+
+    public Integer addition(Integer a, Integer b) {
+        result = a + b;
+        return result;
+    }
+
+    public Integer subtraction(Integer a, Integer b) {
+        result = a - b;
+        return result;
+    }
+
+    public Integer division(Integer a, Integer b) {
+        if (a < b && a != 0) {
+            result = b / a;
+            return result;
+        }
+        if (a > b && b != 0) {
+            result = a / b;
+            return result;
+        }
+        return 1;
+    }
```

I would say it's weird result when you swap the divisor and divider, if I put a / b I want that to have it, but no way I expect b/a

---

Also here you can throw an Arithmetic exception when input is a/0, but it's up to you

---

done